README.md

```diff
@@ -1,25 +1,25 @@
 # Notification and download of Met Office atmospheric model data
-### Timely access to forecast data at a granular level  
+### Timely access to forecast data at a granular level
   This API provides access to UK and Global atmospheric model data to customers who have signed up for a service.
 
 ### Services available:
 
-   - Global Model  
-   - MOGREPS-G Ensemble  
-   - UKV Model  
-   - MOGREPS-UK Ensemble  
+   - Global Model
+   - MOGREPS-G Ensemble
+   - UKV Model
+   - MOGREPS-UK Ensemble
 
 # Prerequisites:
 
   * Sign up to one or more of the Atmospheric Model data services.
   * Receive one or more API keys.
-  * Receive one or more SNS topic Amazon Resource Names (ARNs).
+  * Receive one or more SNS topic Amazon Resource Names (ARNs) if using an AWS accont.
 
 # Actions:
 
-   1. Set up an [endpoint](aws-example/README.md) which can receive SNS notifications.
-  
-   2.  During model output periods, receive notifications of newly available files on the endpoint in a format similar to the following:
+   1. Set up an endpoint which can receive [SNS](aws-example/README.md) notifications or [http](http-endpoint-example/README.md) messages.

```

Comment:
👍 for including both examples here. Both examples process SNS messages though, so perhaps this should be something like:
```
Set up a way to receive SNS messages: either via [SQS](aws-example/README.md) or [HTTP](http-endpoint-example/README.md).
```
NB. In my suggestion I've also changed the capitalisation of `http` to make it `HTTP`.